src/emulator/auth/operations.ts

```diff
@@ -166,6 +166,31 @@ function signUp(
     updates.passwordUpdatedAt = Date.now();
     updates.validSince = toUnixTimestamp(new Date()).toString();
   }
+  if (reqBody.mfaInfo) {
+    const mfaInfo = [];
+    for (const factor of reqBody.mfaInfo) {
+      // the node SDK validation asserts the absence of a uid on create requests prior to submitting any API requests.
```

Comment:
I think production just simply ignores the `mfaEnrollmentId` field and let's just follow that. We can simply just write `mfaInfo.push({displayName: factor.displayName, phoneInfo: factor.phoneInfo, mfaEnrollmentId})` below.